soda/contracts/tests/contracts/contract_execution_test.py

```diff
@@ -0,0 +1,84 @@
+import logging
+from textwrap import dedent
+from unittest import skip
+
+from contracts.data_contract_translator import DataContractTranslator
+from helpers.common_test_tables import customers_test_table
+from helpers.data_source_fixture import DataSourceFixture
+
+logger = logging.getLogger(__name__)
+
+
+@skip
+def test_contract_transformation_and_execution(data_source_fixture: DataSourceFixture):
+    table_name = data_source_fixture.ensure_test_table(customers_test_table)
+
+    source_yaml_string = dedent(
+        f"""
+        dataset: {table_name}
+
+        columns:
+          - name: id
+            data_type: character varying
+            unique: true
+            attributes:
```

Comment:
Attributes? Tell me more! :)

Reply:
These have to be first implemented on Soda Cloud.  Atm, they are not as far as I know.  I will remove them from the code/tests